### ISSUE TITLE
Access to rendering options

### ISSUE THREAD

**Issue by wpmccormick** [sonntam/node-red-contrib-xstate-machine#26]
Based on what I see [here](https://state-machine-cat.js.org/), there are some rendering controls that are not exposed. It would be very useful to control the render direction and the fi to width option. For the latter, It might be sufficient if _fit to width_ was just always on, as in my case, a long skinny chart is difficult to see what's happening here.

**Comment by sonntam** [comment]
Good idea! Maybe there is a way to change the initial SVG viewport configuration to get some kind of „fit to width“?

Unfortunately I currently don‘t have free time to work on the extension. You are invited to try for yourself and submit a PR. I would handle the npmjs package update.
